src/type.js

```diff
@@ -42,6 +42,12 @@ const modifierCallbackMap = {
     keyCode: 93,
     modifierProperty: 'metaKey',
   }),
+  ...createCapsLockModifierCallbackEntries({
```

Comment:
Rather than using a function, maybe it would make more sense to just inline this one with a comment explaining why it can't use `createModifierCallbackEntries`?

Reply:
Done. Sorry about the delay; I missed this comment at first.